airbyte-integrations/connectors/source-metabase/source_metabase/streams.py

```diff
@@ -53,9 +62,100 @@ def path(self, **kwargs) -> str:
         return "dashboard"
 
 
-class Users(MetabaseStream):
+class Databases(MetabaseStream):
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        result = response.json().get("data", [])
+        yield from result
+
+    def path(self, **kwargs) -> str:
+        return "database"
 
-    response_entity = "data"
+
+class DatasetQueryViews(IncrementalMetabaseStream):
+    primary_key = None
+    cursor_field = "viewed_on"
+    http_method = "POST"
+    page_size = 10000
+    page_offset = 0
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        data_result = response.json().get("data", [])
+        rows_result = data_result.get("rows", [])
+        header = [
```

Comment:
If these headers are stream-specific, let's declare them as a class prop, instead.

Reply:
Do you have examples of how that should be done in another connector?

Reply:
```python
class DatasetQueryViews(IncrementalMetabaseStream):
  HEADERS = ["viewed_on", "card_id", ...]
 ```